format/wasm/testdata/core/generate_fqtest.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+d="$( cd "$( dirname "$0" )" && pwd -P )"
+
+RESET=$'\e[0m'
+BOLD=$'\e[1m'
+GREEN=$'\e[0;32m'
+RED=$'\e[0;31m'
+
+tmpdir=$(mktemp -d -t fq-wasm-generate-fqtest.XXXXXXXX)
+trap 'tear_down' 0
+
+result=1
+tear_down() {
+    : "Clean up tmpdir" && {
+        [[ $tmpdir ]] && rm -rf "$tmpdir"
+    }
+
+    : "Report result" && {
+        if [ "$result" -eq 0 ]; then
+            echo
+            echo -e "${GREEN}${BOLD}OK${RESET}"
+            echo
+        else
+            echo
+            echo -e "${RED}${BOLD}FAILED${RESET}"
+            echo
+        fi
+        exit $result
+    }
+}
+
+(
+  cd "$d"
+  rm -f ./*.fqtest
+
+  go build -o "$tmpdir/fq" "$d/../../../../"
+  find "." -name '*.wasm' -print0 | sort -z -n |
+    while IFS= read -r -d '' f; do
+      f=${f#./}
+      of=${f/%.wasm/.fqtest}
+      echo "$f -> $of"
+      echo "$ fq -d wasm dv $f" > "$of"
+      "$tmpdir/fq" -d wasm dv "$f" >>"$of"
+    done
```

Comment:
You can run the fqtest tests in "write" mode and they will write instead of verify the output, ex something like this would rewrite all .fqtest files with current actual output `WRITE_ACTUAL=1 go test -run TestFQTests/wasm ./format/`. So what i usually do it so just generate .fqtest files with `$ fq ...`  lines etc and then run in write mode.

Reply:
Now it uses `WRITE_ACTUAL=1 go test ...` @ https://github.com/wader/fq/pull/383/commits/7401d141e87df00d916c8154b86ab5aa99cd7b0e